{{cookiecutter.project_shortname}}/Pipfile

```diff
@@ -7,7 +7,6 @@ name = "pypi"
 Babel = ">=2.4.0"
 Flask-BabelEx = ">=0.9.3"
 invenio = { version = ">=3.1.0dev20181106", extras = ["base", "metadata", "{{ cookiecutter.database }}", "auth", {%- if cookiecutter.elasticsearch == "5" %} "elasticsearch5", {%- elif cookiecutter.elasticsearch == "6" %} "elasticsearch6" {%- endif %} ]}
-{{ cookiecutter.project_shortname }} = {editable = true, path = "."}
```

Comment:
Removes the final application from Pipfile, so that it can be installed in the final step after with the dependencies has been built.

Reply:
Since the application is not installed anymore, this should be reflected in the `scripts/bootstrap`, i.e. it should run `pipenv run pip install -e .` after the `pipenv install ...`